File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 04/08/2026, 00:21:19
class Solution {
    public int removeDuplicates(int[] nums) {
        int start = 0;
        for(int i = 0; i < nums.length;i++)
        {
            if(i<nums.length-1 && nums[i] == nums[i+1]) continue;
            else {
                nums[start] = nums[i];
                start++;
            }
        }
        return start;
    }
}

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.cpp
// Last updated: 04/08/2026, 00:21:25
class Solution {
public:
    int removeDuplicates(vector<int>& arr) {
        int start=0;
        for(int i=0;i<arr.size();i++){
            if(i+1<arr.size() && arr[i]==arr[i+1]){
                continue;
            }else{
                arr[start]=arr[i];
                start++;
            }
        }
        return start;
    }
};